server/db/index.js

```
@@ -15,37 +15,40 @@ function getRandom(type, max) {
+        return Promise.all([
+          Promise.all(usersSeed.map(user => User.create(user))),
+          Promise.all(challengesSeed.map(chal => Challenge.create(chal))),
+          Promise.all(userchallengeSeed.map(chal => Userchallenge.create(chal))),
+          Promise.all(solutionsSeed.map(sol => Solution.create(sol))),
+          Promise.all(
+            imagesSeed.map(img => Image.create({ connector: img.connector, data: img.data })),
+          ),
+        ]);
+      })
+      .then(([users, challenges, userchallenges, solutions, images]) => {
+        return Promise.all([
+          userchallenges
+            .find(chal => chal.css.includes('circle'))
+            .update({ userId: getRandom(users, 3), challengeId: 1 }),
+          userchallenges
+            .find(chal => chal.css.includes('square'))
+            .update({ userId: getRandom(users, 3), challengeId: 2 }),
+          solutions.find(sol => sol.css.includes('circle')).update({ challengeId: 1 }),
+          solutions.find(sol => sol.css.includes('square')).update({ challengeId: 2 }),
 
-        images.find(img => img.connector === 'challenge-1').update({ challengeId: 1 }),
-        images.find(img => img.connector === 'userchallenge-1').update({ userchallengeId: 1 }),
-        images.find(img => img.connector === 'challenge-2').update({ challengeId: 2 }),
-        images.find(img => img.connector === 'userchallenge-2').update({ userchallengeId: 2 }),
-      ]);
-    })
-    .catch(err => console.log(err));
+          images.find(img => img.connector === 'challenge-1').update({ challengeId: 1 }),
+          images.find(img => img.connector === 'userchallenge-1').update({ userchallengeId: 1 }),
+          images.find(img => img.connector === 'challenge-2').update({ challengeId: 2 }),
+          images.find(img => img.connector === 'userchallenge-2').update({ userchallengeId: 2 }),
+        ]);
+      })
+      // eslint-disable-next-line no-console
+      .catch(err => console.error(err))
+  );
```

In a closer to production app, we should throw an err.